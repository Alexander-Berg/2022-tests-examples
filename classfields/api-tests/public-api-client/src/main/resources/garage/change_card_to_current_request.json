{
  "cardType": "CURRENT_CAR",
  "exCarAdditionalInfo": {
    "reason": "SOLD"
  }
}