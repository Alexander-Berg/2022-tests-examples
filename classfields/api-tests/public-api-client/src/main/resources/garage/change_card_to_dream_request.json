{
  "cardType": "DREAM_CAR",
  "exCarAdditionalInfo": {
    "reason": "SOLD"
  }
}