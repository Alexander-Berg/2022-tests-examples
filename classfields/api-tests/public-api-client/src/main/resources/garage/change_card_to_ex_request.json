{
  "cardType": "EX_CAR",
  "exCarAdditionalInfo": {
    "reason": "SOLD"
  }
}