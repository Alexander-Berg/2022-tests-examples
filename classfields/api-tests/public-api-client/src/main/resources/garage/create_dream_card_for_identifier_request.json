{
  "card_type": "DREAM_CAR",
  "registration_region_id": 203
}