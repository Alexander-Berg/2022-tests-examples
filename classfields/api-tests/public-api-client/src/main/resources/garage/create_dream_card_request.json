{
  "added_manually": true,
  "card": {
    "vehicle_info": {
      "car_info": {
        "mark": "BMW",
        "model": "M5",
        "super_gen_id": "3722944"
      },
      "color": {
        "id": "0000CC"
      }
    },
    "card_type_info": {
      "card_type": "DREAM_CAR"
    }
  },
  "geo_radius": 200,
  "geo_id": [
    213
  ]
}