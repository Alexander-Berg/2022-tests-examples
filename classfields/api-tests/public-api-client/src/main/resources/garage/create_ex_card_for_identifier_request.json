{
  "card_type": "EX_CAR",
  "registration_region_id": 203
}