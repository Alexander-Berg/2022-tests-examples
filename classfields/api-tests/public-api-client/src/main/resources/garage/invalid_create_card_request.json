{
  "card": {
    "vehicle_info": {
      "car_info": {
        "body_type": "ALLROAD_5_DOORS",
        "engine_type": "GASOLINE",
        "transmission": "VARIATOR",
        "drive": "ALL_WHEEL_DRIVE",
        "mark": "MITSUBISHI",
        "model": "OUTLANDER",
        "super_gen_id": "20526645",
        "configuration_id": "20526649",
        "complectation_id": "20526825",
        "steering_wheel": "LEFT"
      },
      "documents": {
        "owners_number": 20,
        "year": 1900,
        "vin": "X9FLXXEEBLES67719"
      }
    }
  }
}