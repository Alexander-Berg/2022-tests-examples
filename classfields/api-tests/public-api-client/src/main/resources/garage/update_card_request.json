{
  "card": {
    "vehicle_info": {
      "car_info": {
        "body_type": "SEDAN",
        "engine_type": "GASOLINE",
        "transmission": "MECHANICAL",
        "drive": "FORWARD_CONTROL",
        "mark": "PEUGEOT",
        "model": "408",
        "super_gen_id": "8350849",
        "configuration_id": "8350853",
        "tech_param_id": "20089520",
        "horse_power": 110
      },
      "documents": {
        "owners_number": 3,
        "year": 2013,
        "vin": "Z8T4DNFUCDM014995"
      }
    }
  }
}