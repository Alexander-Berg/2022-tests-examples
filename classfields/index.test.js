const cardImageGallery = require('./index');
const getValidInteriorPanoramaData = require('./getValidInteriorPanoramaData');

const cloneOfferWithHelpers = require('autoru-frontend/mockData/state/helpers/offer/cloneOfferWithHelpers');

let offer;
beforeEach(() => {
    offer = {
        state: {
            image_urls: [
                { sizes: { '1200x900n': 'img1' } },
                { sizes: { '1200x900n': 'img2' } },
                { sizes: { '1200x900n': 'img3' } },
            ],
        },
    };
});

describe('порядок galleryItems', () => {
    it('должен расположить фотки в том же порядке', () => {
        expect(cardImageGallery(offer).galleryItems).toEqual([
            { sizes: { '1200x900n': 'img1' }, type: 'IMAGE' },
            { sizes: { '1200x900n': 'img2' }, type: 'IMAGE' },
            { sizes: { '1200x900n': 'img3' }, type: 'IMAGE' },
        ]);
    });

    describe('есть панорама', () => {
        beforeEach(() => {
            offer = cloneOfferWithHelpers(offer).withPanoramaExterior();
        });

        it('должен расположить: внешняя панорама, внутренняя панорама, видео, фотки', () => {
            offer = offer.withPanoramaInterior().withYoutubeVideo().value();

            expect(cardImageGallery(offer).galleryItems).toEqual([
                { ...offer.state.external_panorama.published, type: 'PANORAMA_EXTERIOR' },
                getValidInteriorPanoramaData(offer),
                {
                    type: 'VIDEO',
                    provider: 'YOUTUBE',
                    original: '//i.ytimg.com/vi/ytvideo/hqdefault.jpg',
                    thumbnail: '//i.ytimg.com/vi/ytvideo/mqdefault.jpg',
                    video: '//www.youtube.com/embed/ytvideo?enablejsapi=1',
                },
                { sizes: { '1200x900n': 'img1' }, type: 'IMAGE' },
                { sizes: { '1200x900n': 'img2' }, type: 'IMAGE' },
                { sizes: { '1200x900n': 'img3' }, type: 'IMAGE' },
            ]);
        });

        it('должен расположить: панорама, спинкар, видео, фотки', () => {
            offer = offer.withSpincar().withYoutubeVideo().value();

            expect(cardImageGallery(offer).galleryItems).toEqual([
                { ...offer.state.external_panorama.published, type: 'PANORAMA_EXTERIOR' },
                { sizes: { '1200x900n': 'img2' }, spincar: 'spincar_exterior_url', type: 'SPINCAR' },
                {
                    type: 'VIDEO',
                    provider: 'YOUTUBE',
                    original: '//i.ytimg.com/vi/ytvideo/hqdefault.jpg',
                    thumbnail: '//i.ytimg.com/vi/ytvideo/mqdefault.jpg',
                    video: '//www.youtube.com/embed/ytvideo?enablejsapi=1',
                },
                { sizes: { '1200x900n': 'img1' }, type: 'IMAGE' },
                { sizes: { '1200x900n': 'img2' }, type: 'IMAGE' },
                { sizes: { '1200x900n': 'img3' }, type: 'IMAGE' },
            ]);
        });
    });

    describe('нет панорамы', () => {
        beforeEach(() => {
            offer = cloneOfferWithHelpers(offer);
        });

        it('должен расположить: фотка, внутренняя панорама, видео, фотки', () => {
            offer = offer.withPanoramaInterior().withYoutubeVideo().value();

            expect(cardImageGallery(offer).galleryItems).toEqual([
                { sizes: { '1200x900n': 'img1' }, type: 'IMAGE' },
                getValidInteriorPanoramaData(offer),
                {
                    type: 'VIDEO',
                    provider: 'YOUTUBE',
                    original: '//i.ytimg.com/vi/ytvideo/hqdefault.jpg',
                    thumbnail: '//i.ytimg.com/vi/ytvideo/mqdefault.jpg',
                    video: '//www.youtube.com/embed/ytvideo?enablejsapi=1',
                },
                { sizes: { '1200x900n': 'img2' }, type: 'IMAGE' },
                { sizes: { '1200x900n': 'img3' }, type: 'IMAGE' },
            ]);
        });

        it('должен расположить: фотка, видео, спинкар, фотки', () => {
            offer = offer.withSpincar().withYoutubeVideo().value();

            expect(cardImageGallery(offer).galleryItems).toEqual([
                { sizes: { '1200x900n': 'img1' }, type: 'IMAGE' },
                {
                    type: 'VIDEO',
                    provider: 'YOUTUBE',
                    original: '//i.ytimg.com/vi/ytvideo/hqdefault.jpg',
                    thumbnail: '//i.ytimg.com/vi/ytvideo/mqdefault.jpg',
                    video: '//www.youtube.com/embed/ytvideo?enablejsapi=1',
                },
                { sizes: { '1200x900n': 'img2' }, spincar: 'spincar_exterior_url', type: 'SPINCAR' },
                { sizes: { '1200x900n': 'img2' }, type: 'IMAGE' },
                { sizes: { '1200x900n': 'img3' }, type: 'IMAGE' },
            ]);
        });
    });

    describe('бейджи', () => {
        const offerWithOrigImg = {
            state: {
                image_urls: [
                    { sizes: { '1200x900n': 'img1' } },
                    { sizes: { '1200x900n': 'img2' }, is_hd: true },
                    { sizes: { '1200x900n': 'img3' } },
                ],
            },
        };

        it('должен добавить HD', () => {
            offer = cloneOfferWithHelpers(offerWithOrigImg).withStatus('ACTIVE').value();
            offer.documents = { vin_resolution: 'OK' };

            expect(cardImageGallery(offer).badges.hd).toBeTruthy();
        });

        it('не должен добавить HD, если у оффера нет отчёта', () => {
            offer = cloneOfferWithHelpers(offerWithOrigImg).value();

            expect(cardImageGallery(offer).badges.hd).toBeFalsy();
        });

        it('не должен добавить HD, если у оффера нет hd-фоток', () => {
            offer = cloneOfferWithHelpers(offer).withTags([ 'vin_offers_history' ]).value();

            expect(cardImageGallery(offer).badges.hd).toBeFalsy();
        });
    });
});
