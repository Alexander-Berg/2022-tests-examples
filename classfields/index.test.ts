import { InsuranceType, InsuranceStatus } from '@vertis/schema-registry/ts-types-snake/auto/api/vin/vin_report_model';
import { InsuranceSource } from '@vertis/schema-registry/ts-types-snake/auto/api/vin/garage/garage_api_model';

import getFormattedMoscowDate from 'auto-core/lib/util/time/getFormattedMoscowDate';

import { nbsp } from 'auto-core/react/lib/html-entities';

import dayjs from 'auto-core/dayjs';

import { getInsuranceData, getDaysTo } from './index';

Date.now = jest.fn(() => 1633640055424);

it('getDaysTo возвращает коректное количество дней', () => {
    const nowM10d = dayjs().add(-10, 'd');
    const nowP3d = dayjs().add(3 * 24 + 1, 'h');
    const result = getDaysTo({ from: nowM10d.toISOString(), to: nowP3d.toISOString() });

    expect(result).toBe('3 дня');
});

it('getDaysTo возвращает 5 дней', () => {
    const nowM10d = dayjs().add(-10, 'd');
    const nowP5d = dayjs().add(5 * 24 + 1, 'h');
    const result = getDaysTo({
        from: nowM10d.toISOString(),
        to: nowP5d.toISOString(),
    });

    expect(result).toBe('5 дней');
});

it('getDaysTo возвращает истёк', () => {
    const nowM1D = dayjs().add(-1, 'd');
    const nowM3d = dayjs().add(-365, 'd');
    const result = getDaysTo({ from: nowM3d.toISOString(), to: nowM1D.toISOString() });

    expect(result).toBe('истёк');
});

it('getDaysTo не возвращает 0', () => {
    const now = dayjs();
    const nowM3d = dayjs().add(-365, 'd');
    const result = getDaysTo({ from: nowM3d.toISOString(), to: now.toISOString() });

    expect(result).toBe('1 день');
});

it('getInsuranceData возвращает минимум 2 поля: "Начало действия" и "Серия и номер"', () => {
    const result = getInsuranceData({
        insurance_type: InsuranceType.OSAGO,
        is_actual: true,
        number: '111',
        serial: '',
        status: InsuranceStatus.ACTIVE,
        source: InsuranceSource.UNKNOWN_SOURCE,
    });

    expect(result).toMatchSnapshot();
});

it('getInsuranceData возвращает "Окончание действия"', () => {
    const now = dayjs();
    const nowM1Y = dayjs().add(-365, 'd');
    const nowM1D = dayjs().add(-1, 'd');
    const props: { [key: string]: string | boolean | undefined } = {};

    getInsuranceData({
        serial: '',
        number: '111',
        insurance_type: InsuranceType.OSAGO,
        status: InsuranceStatus.ACTIVE,
        from: nowM1Y.toISOString(),
        is_actual: true,
        source: InsuranceSource.UNKNOWN_SOURCE,
    }).forEach(x => (props[x.key] = x.value));

    expect(props.insuranceTo).toBe(getFormattedMoscowDate(now).replace(/\s/g, nbsp));
    expect(props.serialNumber).toBe('111');

    getInsuranceData({
        serial: 'XXX',
        number: '111',
        insurance_type: InsuranceType.KASKO,
        status: InsuranceStatus.ACTIVE,
        from: nowM1Y.toISOString(),
        to: nowM1D.toISOString(),
        is_actual: true,
        source: InsuranceSource.UNKNOWN_SOURCE,
    }).forEach(x => (props[x.key] = x.value));

    expect(props.insuranceTo).toBe(getFormattedMoscowDate(nowM1D).replace(/\s/g, nbsp));
    expect(props.insuranceFrom).toBe(getFormattedMoscowDate(nowM1Y).replace(/\s/g, nbsp));
    expect(props.serialNumber).toBe('XXX 111');

});
