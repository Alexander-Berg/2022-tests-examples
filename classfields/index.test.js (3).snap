// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`–ì—Ä—É–ø–ø—ã –∫–∞—Ä—Ç–æ—á–µ–∫ –¥–æ–ª–∂–µ–Ω –≤–µ—Ä–Ω—É—Ç—å –ø—Ä–∞–≤–∏–ª—å–Ω—ã–µ –ø–æ–ª—è –¥–ª—è –≥—Ä—É–ø–ø—ã c –æ—Ñ—Ñ–µ—Ä–∞–º–∏ 1`] = `
Object {
  "amphtml": "https://autoru_frontend.base_domain/moskva/amp/cars/new/group/audi/q3/20293979-20294010/",
  "canonical": "https://autoru_frontend.base_domain/moskva/cars/new/group/audi/q3/20293979-20294010/",
  "description": "–ù–æ–≤—ã–π Audi Q3 I –†–µ—Å—Ç–∞–π–ª–∏–Ω–≥, –≤–Ω–µ–¥–æ—Ä–æ–∂–Ω–∏–∫ 5-–¥–≤–µ—Ä–Ω—ã–π –≤ –ú–æ—Å–∫–≤–µ –Ω–∞ –ê–≤—Ç–æ.—Ä—É",
  "image": "https://images.mds-proxy.test.avto.ru/get-autoru-vos/2071343/df0f08c6c9f772cf4486964f4c00e2f6/1200x900",
  "ogTitle": "–ö—É–ø–∏—Ç—å –Ω–æ–≤—ã–π Audi Q3 I –†–µ—Å—Ç–∞–π–ª–∏–Ω–≥ –í–Ω–µ–¥–æ—Ä–æ–∂–Ω–∏–∫ 5 –¥–≤. –ê–≤—Ç–æ.—Ä—É",
  "structuredData": Array [
    Object {
      "@context": "https://schema.org",
      "@type": "https://schema.org/Product",
      "brand": undefined,
      "color": undefined,
      "description": undefined,
      "image": Object {
        "@context": "https://schema.org",
        "@type": "ImageObject",
        "author": "",
        "contentUrl": undefined,
        "datePublished": "",
        "description": "  undefined —Ü–≤–µ—Ç–∞ –ø–æ —Ü–µ–Ω–µ  –≤ undefined",
        "name": "  undefined —Ü–≤–µ—Ç–∞",
      },
      "name": " ",
      "offers": Object {
        "@context": "https://schema.org",
        "@type": "AggregateOffer",
        "highPrice": undefined,
        "lowPrice": 0,
        "offerCount": undefined,
        "priceCurrency": "RUB",
      },
      "productionDate": undefined,
    },
    Object {
      "@context": "https://schema.org",
      "@type": "http://schema.org/BreadcrumbList",
      "itemListElement": Array [
        Object {
          "@type": "ListItem",
          "item": Object {
            "@id": "https://autoru_frontend.base_domain/cars/audi/new/?do_not_redirect=true",
            "@type": "Thing",
          },
          "name": "Audi",
          "position": 1,
        },
        Object {
          "@type": "ListItem",
          "item": Object {
            "@id": "https://autoru_frontend.base_domain/cars/audi/q3/new/?do_not_redirect=true",
            "@type": "Thing",
          },
          "name": "Q3",
          "position": 2,
        },
      ],
    },
    Object {
      "@context": "https://schema.org",
      "@type": "SaleEvent",
      "description": "–°—Ä–∞–≤–Ω–∏ —Å—Ç–æ–∏–º–æ—Å—Ç—å –û–°–ê–ì–û —É —Ä–∞–∑–Ω—ã—Ö –∫–æ–º–ø–∞–Ω–∏–π",
      "endDate": "2022-02-01T21:00:00+03:00",
      "location": Object {
        "@type": "Place",
        "name": "–ê–≤—Ç–æ.—Ä—É",
      },
      "name": "üîî –°—Ç—Ä–∞—Ö–æ–≤–∫–∞-–æ–Ω–ª–∞–π–Ω",
      "startDate": "2021-12-01T21:00:00+03:00",
    },
    Object {
      "@context": "https://schema.org",
      "@type": "SaleEvent",
      "description": "–ü—Ä–æ–≤–µ—Ä–∫–∞ –ü—Ä–æ–ê–≤—Ç–æ",
      "endDate": "2022-02-01T21:00:00+03:00",
      "location": Object {
        "@type": "Place",
        "name": "–ê–≤—Ç–æ.—Ä—É",
      },
      "name": "üîî –ü—Ä–æ–≤–µ—Ä–∫–∞ –ø–æ VIN",
      "startDate": "2021-12-01T21:00:00+03:00",
    },
    Object {
      "@context": "https://schema.org",
      "@type": "SaleEvent",
      "description": "–î–æ–±–∞–≤—å —Å–≤–æ–µ –∞–≤—Ç–æ –≤ –ì–∞—Ä–∞–∂ –∏ –ø–æ–ª—É—á–∞–π –±–æ–Ω—É—Å—ã",
      "endDate": "2022-02-01T21:00:00+03:00",
      "location": Object {
        "@type": "Place",
        "name": "–ê–≤—Ç–æ.—Ä—É",
      },
      "name": "üîî –°–∫–∏–¥–∫–∏ –≤ —Ç–≤–æ–µ–º –ì–∞—Ä–∞–∂–µ",
      "startDate": "2021-12-01T21:00:00+03:00",
    },
  ],
  "title": "–ö—É–ø–∏—Ç—å –Ω–æ–≤—ã–π Audi Q3 I –†–µ—Å—Ç–∞–π–ª–∏–Ω–≥ –≤ –ú–æ—Å–∫–≤–µ | –¶–µ–Ω—ã –Ω–∞ –ê—É–¥–∏ –ö—É3 I –†–µ—Å—Ç–∞–π–ª–∏–Ω–≥ –≤–Ω–µ–¥–æ—Ä–æ–∂–Ω–∏–∫ 5-–¥–≤–µ—Ä–Ω—ã–π –Ω–∞ –ê–≤—Ç–æ.—Ä—É",
}
`;

exports[`–ì—Ä—É–ø–ø—ã –∫–∞—Ä—Ç–æ—á–µ–∫ –¥–æ–ª–∂–µ–Ω –≤–µ—Ä–Ω—É—Ç—å –ø—Ä–∞–≤–∏–ª—å–Ω—ã–µ –ø–æ–ª—è –¥–ª—è –≥—Ä—É–ø–ø—ã –±–µ–∑ –æ—Ñ—Ñ–µ—Ä–æ–≤ 1`] = `
Object {
  "amphtml": "https://autoru_frontend.base_domain/moskva/amp/cars/new/group/audi/q3/20293979-20294010/",
  "canonical": "https://autoru_frontend.base_domain/moskva/cars/new/group/audi/q3/20293979-20294010/",
  "description": "–ù–æ–≤—ã–π Audi Q3 I –†–µ—Å—Ç–∞–π–ª–∏–Ω–≥, –≤–Ω–µ–¥–æ—Ä–æ–∂–Ω–∏–∫ 5-–¥–≤–µ—Ä–Ω—ã–π –≤ –ú–æ—Å–∫–≤–µ –Ω–∞ –ê–≤—Ç–æ.—Ä—É",
  "image": "https://avatars.mds.yandex.net/get-verba/937147/2a0000016dcf95343f30126207cd576076e2/cattouch",
  "ogTitle": "–ö—É–ø–∏—Ç—å –Ω–æ–≤—ã–π Audi Q3 I –†–µ—Å—Ç–∞–π–ª–∏–Ω–≥ –í–Ω–µ–¥–æ—Ä–æ–∂–Ω–∏–∫ 5 –¥–≤. –ê–≤—Ç–æ.—Ä—É",
  "structuredData": Array [
    null,
    Object {
      "@context": "https://schema.org",
      "@type": "http://schema.org/BreadcrumbList",
      "itemListElement": Array [
        Object {
          "@type": "ListItem",
          "item": Object {
            "@id": "https://autoru_frontend.base_domain/cars/audi/new/?do_not_redirect=true",
            "@type": "Thing",
          },
          "name": "Audi",
          "position": 1,
        },
        Object {
          "@type": "ListItem",
          "item": Object {
            "@id": "https://autoru_frontend.base_domain/cars/audi/q3/new/?do_not_redirect=true",
            "@type": "Thing",
          },
          "name": "Q3",
          "position": 2,
        },
      ],
    },
    Object {
      "@context": "https://schema.org",
      "@type": "SaleEvent",
      "description": "–°—Ä–∞–≤–Ω–∏ —Å—Ç–æ–∏–º–æ—Å—Ç—å –û–°–ê–ì–û —É —Ä–∞–∑–Ω—ã—Ö –∫–æ–º–ø–∞–Ω–∏–π",
      "endDate": "2022-02-01T21:00:00+03:00",
      "location": Object {
        "@type": "Place",
        "name": "–ê–≤—Ç–æ.—Ä—É",
      },
      "name": "üîî –°—Ç—Ä–∞—Ö–æ–≤–∫–∞-–æ–Ω–ª–∞–π–Ω",
      "startDate": "2021-12-01T21:00:00+03:00",
    },
    Object {
      "@context": "https://schema.org",
      "@type": "SaleEvent",
      "description": "–ü—Ä–æ–≤–µ—Ä–∫–∞ –ü—Ä–æ–ê–≤—Ç–æ",
      "endDate": "2022-02-01T21:00:00+03:00",
      "location": Object {
        "@type": "Place",
        "name": "–ê–≤—Ç–æ.—Ä—É",
      },
      "name": "üîî –ü—Ä–æ–≤–µ—Ä–∫–∞ –ø–æ VIN",
      "startDate": "2021-12-01T21:00:00+03:00",
    },
    Object {
      "@context": "https://schema.org",
      "@type": "SaleEvent",
      "description": "–î–æ–±–∞–≤—å —Å–≤–æ–µ –∞–≤—Ç–æ –≤ –ì–∞—Ä–∞–∂ –∏ –ø–æ–ª—É—á–∞–π –±–æ–Ω—É—Å—ã",
      "endDate": "2022-02-01T21:00:00+03:00",
      "location": Object {
        "@type": "Place",
        "name": "–ê–≤—Ç–æ.—Ä—É",
      },
      "name": "üîî –°–∫–∏–¥–∫–∏ –≤ —Ç–≤–æ–µ–º –ì–∞—Ä–∞–∂–µ",
      "startDate": "2021-12-01T21:00:00+03:00",
    },
  ],
  "title": "–ö—É–ø–∏—Ç—å –Ω–æ–≤—ã–π Audi Q3 I –†–µ—Å—Ç–∞–π–ª–∏–Ω–≥ –≤ –ú–æ—Å–∫–≤–µ | –¶–µ–Ω—ã –Ω–∞ –ê—É–¥–∏ –ö—É3 I –†–µ—Å—Ç–∞–π–ª–∏–Ω–≥ –≤–Ω–µ–¥–æ—Ä–æ–∂–Ω–∏–∫ 5-–¥–≤–µ—Ä–Ω—ã–π –Ω–∞ –ê–≤—Ç–æ.—Ä—É",
}
`;

exports[`–ì—Ä—É–ø–ø—ã –∫–∞—Ä—Ç–æ—á–µ–∫ –¥–æ–ª–∂–µ–Ω –≤–µ—Ä–Ω—É—Ç—å –ø—Ä–∞–≤–∏–ª—å–Ω—ã–µ –ø–æ–ª—è –¥–ª—è –≥—Ä—É–ø–ø—ã –±–µ–∑ –æ—Ñ—Ñ–µ—Ä–æ–≤ –∏ –±–µ–∑ —Ä–µ–Ω–¥–µ—Ä–æ–≤ 1`] = `
Object {
  "amphtml": "https://autoru_frontend.base_domain/moskva/amp/cars/new/group/audi/q3/20293979-20294010/",
  "canonical": "https://autoru_frontend.base_domain/moskva/cars/new/group/audi/q3/20293979-20294010/",
  "description": "–ù–æ–≤—ã–π Audi Q3 I –†–µ—Å—Ç–∞–π–ª–∏–Ω–≥, –≤–Ω–µ–¥–æ—Ä–æ–∂–Ω–∏–∫ 5-–¥–≤–µ—Ä–Ω—ã–π –≤ –ú–æ—Å–∫–≤–µ –Ω–∞ –ê–≤—Ç–æ.—Ä—É",
  "image": "https://avatars.mds.yandex.net/get-verba/787013/2a0000016095f2130bbdf48f5215b874d157/cattouch",
  "ogTitle": "–ö—É–ø–∏—Ç—å –Ω–æ–≤—ã–π Audi Q3 I –†–µ—Å—Ç–∞–π–ª–∏–Ω–≥ –í–Ω–µ–¥–æ—Ä–æ–∂–Ω–∏–∫ 5 –¥–≤. –ê–≤—Ç–æ.—Ä—É",
  "structuredData": Array [
    null,
    Object {
      "@context": "https://schema.org",
      "@type": "http://schema.org/BreadcrumbList",
      "itemListElement": Array [
        Object {
          "@type": "ListItem",
          "item": Object {
            "@id": "https://autoru_frontend.base_domain/cars/audi/new/?do_not_redirect=true",
            "@type": "Thing",
          },
          "name": "Audi",
          "position": 1,
        },
        Object {
          "@type": "ListItem",
          "item": Object {
            "@id": "https://autoru_frontend.base_domain/cars/audi/q3/new/?do_not_redirect=true",
            "@type": "Thing",
          },
          "name": "Q3",
          "position": 2,
        },
      ],
    },
    Object {
      "@context": "https://schema.org",
      "@type": "SaleEvent",
      "description": "–°—Ä–∞–≤–Ω–∏ —Å—Ç–æ–∏–º–æ—Å—Ç—å –û–°–ê–ì–û —É —Ä–∞–∑–Ω—ã—Ö –∫–æ–º–ø–∞–Ω–∏–π",
      "endDate": "2022-02-01T21:00:00+03:00",
      "location": Object {
        "@type": "Place",
        "name": "–ê–≤—Ç–æ.—Ä—É",
      },
      "name": "üîî –°—Ç—Ä–∞—Ö–æ–≤–∫–∞-–æ–Ω–ª–∞–π–Ω",
      "startDate": "2021-12-01T21:00:00+03:00",
    },
    Object {
      "@context": "https://schema.org",
      "@type": "SaleEvent",
      "description": "–ü—Ä–æ–≤–µ—Ä–∫–∞ –ü—Ä–æ–ê–≤—Ç–æ",
      "endDate": "2022-02-01T21:00:00+03:00",
      "location": Object {
        "@type": "Place",
        "name": "–ê–≤—Ç–æ.—Ä—É",
      },
      "name": "üîî –ü—Ä–æ–≤–µ—Ä–∫–∞ –ø–æ VIN",
      "startDate": "2021-12-01T21:00:00+03:00",
    },
    Object {
      "@context": "https://schema.org",
      "@type": "SaleEvent",
      "description": "–î–æ–±–∞–≤—å —Å–≤–æ–µ –∞–≤—Ç–æ –≤ –ì–∞—Ä–∞–∂ –∏ –ø–æ–ª—É—á–∞–π –±–æ–Ω—É—Å—ã",
      "endDate": "2022-02-01T21:00:00+03:00",
      "location": Object {
        "@type": "Place",
        "name": "–ê–≤—Ç–æ.—Ä—É",
      },
      "name": "üîî –°–∫–∏–¥–∫–∏ –≤ —Ç–≤–æ–µ–º –ì–∞—Ä–∞–∂–µ",
      "startDate": "2021-12-01T21:00:00+03:00",
    },
  ],
  "title": "–ö—É–ø–∏—Ç—å –Ω–æ–≤—ã–π Audi Q3 I –†–µ—Å—Ç–∞–π–ª–∏–Ω–≥ –≤ –ú–æ—Å–∫–≤–µ | –¶–µ–Ω—ã –Ω–∞ –ê—É–¥–∏ –ö—É3 I –†–µ—Å—Ç–∞–π–ª–∏–Ω–≥ –≤–Ω–µ–¥–æ—Ä–æ–∂–Ω–∏–∫ 5-–¥–≤–µ—Ä–Ω—ã–π –Ω–∞ –ê–≤—Ç–æ.—Ä—É",
}
`;
